packages/perseus/src/widgets/radio/radio.ff.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+
+import RadioOld from "./radio-component";
+import RadioNew from "./radio-component.new";
+
+import type {RenderProps} from "./radio-component";
+import type {WidgetProps} from "../../types";
+import type {PerseusRadioRubric} from "@khanacademy/perseus-core";
+
+type Props = WidgetProps<RenderProps, PerseusRadioRubric>;
+
+/**
+ * This is a wrapper around the old radio widget that allows us to
+ * conditionally render the new radio widget when the feature flag is on.
+ *
+ * This is necessary to ensure that we do not interrupt the assessment studies
+ * that are currently running.
+ *
+ * TODO(LEMS-2994): Clean up this file.
+ */
+class Radio extends RadioOld {
+    ffIsOn = false;
+    radioRef = React.createRef<RadioOld | RadioNew>();
+
+    constructor(props: Props) {
+        super(props);
+        this.ffIsOn =
+            props.apiOptions.flags?.["new-radio-widget"] ?? this.ffIsOn;
```

Comment:
what do you think of being explicit here?

```suggestion
            props.apiOptions.flags?.["new-radio-widget"] ?? false;
```